h/templates/client/viewer.html

```diff
@@ -1,15 +1,17 @@
-<selection-tabs ng-if="feature('selection_tabs') && !search.query() && selectedAnnotationCount() === 0"
+<selection-tabs ng-if="feature('selection_tabs') && !search.query() && selectedAnnotationCount() === 0 && !waitingToAnchorAnnotations"
```

Comment:
Reading lines of more than about 100 characters in GitHub's diff viewer is annoying. If a line in a template goes beyond about 80 chars then I would recommend wrapping the attributes onto separate lines as is done below.